code/game/gamemodes/cult/runes.dm

```diff
@@ -167,11 +167,12 @@ structure_check() searches for nearby cultist structures required for the invoca
 /obj/effect/rune/proc/burn_invokers(var/list/mobstoburn)
 	for(var/M in mobstoburn)
 		var/mob/living/L = M
-		to_chat(L, "<span class='cultlarge'><i>\"YOUR SOUL BURNS WITH YOUR ARROGANCE!!!\"</i></span>")
-		if(L.reagents)
-			L.reagents.add_reagent("hell_water", 10)
-		L.Weaken(7)
-		L.Stun(7)
+		if(L)
```

Comment:
Not sure how a null entry would get in this list

Reply:
No clue either. It did happen, though, got the runtime from the public runtime logs of the live server

Reply:
It's because of this line https://github.com/ParadiseSS13/Paradise/blob/49bdfbf3f02b91b56be4a1bdface2d51f6544dcf/code/game/gamemodes/cult/runes.dm#L147
if List len == req_cultists
then it'll pick req_cultists + 1 cultists from that list. And if the list is empty it'll put a null in there. Change it from 0 to 1 and it'll work. No need for this if here then